package.json

```diff
@@ -73,7 +73,7 @@
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "3.0.0",
     "ember-cli": "~4.8.0",
-    "ember-cli-addon-docs": "^5.1.0",
```

Comment:
This version of `ember-cli-addon-docs` use the older (2.1.2) version of ember concurrency